index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img id="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <a href=""
+        ><img
+          id="nav-logo"
+          src="./images/header/Hamburger_icon.svg.png"
+          alt="logo"
+      /></a>
+    </header>
+    <main id="headline">
+      <section>
+        <div id="name"><p>I’m Ali Hassan. Glad to see you!</p></div>
```

Comment:
- [ ] Following the best practices, this is the main title of your website and it should be an `h1` tag. I suggest you update this from the `p` tag to `h1` :smile: :100: